Hearthstone Deck Tracker/Hearthstone/DungeonRun.cs

```diff
@@ -22,12 +22,24 @@ public static Deck GetDefaultDeck(string playerClass, CardSet set)
 				IsDungeonDeck = true,
 				LastEdited = DateTime.Now
 			};
-			//Hack to avoid multiple templates. Should be good enough 99.9% of the time.
-			var template = Config.Instance.DungeonRunDeckNameTemplate.Replace("Dungeon Run", "Monster Hunt");
+			var template = getDeckTemplate(set);
```

Comment:
This needs to more gracefully handle the case where `GetDeckTemplate` returns null (likely in the case of a new set release with a dungeon run type mode).

Reply:
Definitely! I'm planning to make `GetDefaultDeck` return null in that case, since the one reference from `DeckManager` already handles that